<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrdenesServicio</class>
 <widget class="QMainWindow" name="OrdenesServicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2122</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ordenes de servicio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ordenes de servicio</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_vehiculos">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrar Orden</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolver">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>620</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Descripcion">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Introduce una descripcion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Observaciones">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>290</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Introduce una observacion</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="FechaIngreso">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>380</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAsignar">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>520</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Asignar orden</string>
    </property>
   </widget>
   <widget class="QWidget" name="imagen" native="true">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>60</y>
      <width>411</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2122</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
